### ISSUE TITLE
The user principal with name 'AosPrincipalUser' is invalid

### ISSUE THREAD

**Issue by fordummie** [MicrosoftDocs/dynamics-365-unified-operations-public#415]
On "LocalAgentCLI.exe Install"  command anyone? (19 step)

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: d86d2857-6199-f3cb-208b-db4e1cd9f8e2
* Version Independent ID: d425ad9e-6c01-6145-dad5-2af32ffaaff0
* Content: [Troubleshoot on-premises deployments - Finance & Operations | Dynamics 365 | #MSDyn365FO](https://docs.microsoft.com/en-us/dynamics365/unified-operations/dev-itpro/deployment/troubleshoot-on-prem#stateful-and-stateless-services-how-to-identify-nodes-for-log-review)
* Content Source: [articles/dev-itpro/deployment/troubleshoot-on-prem.md](https://github.com/MicrosoftDocs/Dynamics-365-Unified-Operations-Public/blob/live/articles/dev-itpro/deployment/troubleshoot-on-prem.md)
* Service: **dynamics-ax-platform**
* GitHub Login: @sarvanisathish
* Microsoft Alias: **sarvanis**

**Comment by tonyafehr** [comment]
Can you provide more information about your question. Thanks!

**Comment by fordummie** [comment]
Hi @tonyafehr I discovered that in this environment the svc-LocalAgent account of the domain account type did not exist. Only the svc-LocalAgent account of type GMSA. The strangest thing is that the installation manual prompts you to create this account name with the GMSA type and not the domain account. I created the domain account svc-LocalAgent and another account of type GMSA with the name gmsaLocalAgent and I had no more problems.
